geometry_msgs/PoseStamped pose
std_msgs/Float32 height_status